internal/pkg/cli/env_init_test.go

```diff
@@ -107,13 +107,11 @@ func TestEnvAdd_Execute(t *testing.T) {
 				prog:          mockSpinner,
 			},
 			expectedEnv: archer.Environment{
-				Name:    "env",
-				Project: "project",
-				//TODO update these to real values
-				AccountID:          "1234",
-				Region:             "1234",
-				Prod:               true,
-				PublicLoadBalancer: true,
+				Name:      "env",
+				Project:   "project",
+				AccountID: "1234",
+				Region:    "1234",
+				Prod:      true,
```

Comment:
seems like `RegistryURL` is missing? Same for a few other places that create `archer.Environment` below.

Reply:
updated :D